roles/ceph-client/defaults/main.yml

```diff
@@ -7,8 +7,8 @@ fetch_directory: fetch/
 
 user_config: false
 pools:
-  - { name: test, pgs: "{{ pool_default_pg_num }}" }
-  - { name: test2, pgs: "{{ pool_default_pg_num }}" }
+  - { name: test, pgs: "{{ ceph_conf_overrides.global.osd_pool_default_pg_num }}" }
+  - { name: test2, pgs: "{{ ceph_conf_overrides.global.osd_pool_default_pg_num }}" }
```

Comment:
Not really, I mean these will be called only if those are valid https://github.com/ceph/ceph-ansible/blob/e3fb818c663383bd987c4a29cdde155c58ac808d/roles/ceph-client/tasks/main.yml#L5-L6

If not declared the creation task is skipped and the vars never called.

Am I getting this right?